vnext/PropertySheets/External/Microsoft.ReactNative.CppApp.props

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We are bound to get 'shared' properties between the languages, we should consider adding a "shared" set of targets.

In the nuget world this would be props and targets in each pacakges of the helpers: `Microsoft.ReactNative.Uwp` and `Microsoft.ReactNative.Uwp.Managed`, but Microsoft.ReactNative.dll should be it's own nuget package with it's own props and targets as needed. 

Just thinking ahead. Totally fine to do later, when we get our first shared property.

Reply:
wrt package references: the links you provided clearly say C++ is not supported, people want it, and the VS team isn't going to do it

wrt shared targets: I am specifically not pre-optimizing for sharing, because the more I actually try the scenarios, (building apps, testing dependency upgrades, etc) the more changes I've had to make. I *do not* want to propagate the confusing absurdity of PropertySheets we have currently under `vnext/PropertySheets`. That's why I created a new External folder single files.